Client/Frontend/Strings.swift

```diff
@@ -797,3 +797,14 @@ extension Strings {
 extension String {
     public static let FxAQRCode_Instructions = NSLocalizedString("fxa.qr-scanning-view.instructions", value: "Scan the QR code shown at firefox.com/pair", comment: "Instructions shown on qr code scanning view")
 }
+
+//Today Widget Strings - [New Search - Private Search]
+extension String {
+    public static let NewPrivateTabButtonLabel = NSLocalizedString("TodayWidget.PrivateTabButtonLabelV1", tableName: "Today", value: "Private Search", comment: "Open New Private Tab button label")
```

Comment:
All strings are merged on main with some refactoring so we should find a good way to rebase this branch with main